tests/lmeval/test_lmeval.py

```diff
@@ -34,11 +36,15 @@ class LmEvalConfig(BaseModel):
     logger.warning("lm_eval is not installed. This test will be skipped")
 
 TEST_DATA_FILE = os.environ.get("TEST_DATA_FILE", None)
+TIMINGS_DIR = os.environ.get("TIMINGS_DIR", "timings/lm-eval")
 
 
 # Will run each test case in its own process through run_tests.sh
 # emulating vLLM CI testing
 @requires_gpu_count(1)
+@pytest.mark.parametrize(
+    "test_data_file", [pytest.param(TEST_DATA_FILE, id=TEST_DATA_FILE)]
+)
```

Comment:
To be consistent with `test_vllm`:
```suggestion
@parameterized_class("test_data_file", [(TEST_DATA_FILE,)])
```